src/3d-force-graph.js

```diff
@@ -127,6 +127,10 @@ export default Kapsule({
       }
       return this;
     },
+    startAnimation: function(state) {
+      state.animate();
```

Comment:
It's important to check here whether the animation is already running, otherwise this will generate multiple concurrent rendering cycles.